images/pmd_cpd/Dockerfile.erb

```
@@ -0,0 +1,13 @@
+<%= render_erb 'images/Dockerfile.java.erb' %>
+<%= render_erb 'images/Dockerfile.base.erb' %>
+
+COPY --chown=<%= chown %> images/pmd_cpd/cpd ${RUNNER_USER_HOME}/bin/
+
+# NOTE: PMD does not have a CLI option to show its version.
+COPY --chown=<%= chown %> images/pmd_cpd/ShowPMDVersion.java ${RUNNER_USER_HOME}/bin/
+COPY --chown=<%= chown %> images/pmd_cpd/show_pmd_version ${RUNNER_USER_HOME}/bin/
```

[suggestion] How about borrowing the files in `images/pmd_java` as follows?

```suggestion
COPY --chown=<%= chown %> images/pmd_java/ShowPMDVersion.java ${RUNNER_USER_HOME}/bin/
COPY --chown=<%= chown %> images/pmd_java/show_pmd_version ${RUNNER_USER_HOME}/bin/
```

If doing so, `images/pmd_cpd/{show_pmd_version,ShowPMDVersion.java}` can be removed.

---

Good call!
Fixed. commit ffdbdeeec9214f7f9518a60e0bb4b3f5c2befaa3